src/services/ai-server/ai-persona-service/dto/ai.persona.service.dto.update.ts

```
@@ -1,17 +1,20 @@
 import { Field, InputType } from '@nestjs/graphql';
 import { MaxLength } from 'class-validator';
-import { LONG_TEXT_LENGTH, SMALL_TEXT_LENGTH } from '@src/common/constants';
+import { SMALL_TEXT_LENGTH } from '@src/common/constants';
 import JSON from 'graphql-type-json';
```

Biome

[error] Do not shadow the global "JSON" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)